examples/technical_blogs/tartan_blog/build_and_run_networks.py

```diff
@@ -0,0 +1,156 @@
+import subprocess
+import os
+import shutil
+import requests
+
+# Prereqs: portBLAS, SYCL-DNN, oneapi oneapi_construction_kit should be cloned and installed.
+portBLAS_source = os.environ.get("portBLAS_source")
+SYCL_DNN_source = os.environ.get("SYCL-DNN_source")
+path_to_oneapi = os.environ.get("path_to_oneapi")
+OCK_libCL_path = os.environ.get("OCK_libCL_path")
+input_image = os.environ.get("input_image")
+
+# Set environment variables
+os.environ["CMAKE_CXX_COMPILER"] = os.path.join(path_to_oneapi, "compiler/linux/bin-llvm/clang++")
+os.environ["LD_LIBRARY_PATH"] = f"{os.path.join(path_to_oneapi, 'compiler/linux/lib/')}:{os.path.join(path_to_oneapi, 'compiler/linux/compiler/lib/intel64_lin')}:{os.environ.get('LD_LIBRARY_PATH', '')}"
+os.environ["CXX"] = os.environ["CMAKE_CXX_COMPILER"]
+os.environ["CC"] = os.path.join(path_to_oneapi, "compiler/linux/bin-llvm/clang")
+
+def create_directory(path):
+    if os.path.exists(path):
+        shutil.rmtree(path)
+        print(f"Directory '{path}' removed successfully.")
+    os.mkdir(path)
+    print(f"Directory '{path}' created successfully.")
+
+def run_cmake_and_ninja(cmake_command):
```

Comment:
I think this could take the build directory as a parameter, and we invoke `cmake -B f{build_dir}` && `ninja -C f{build_dir}`.

Then people can use this function "anywhere" and don't have to `cd` into the build directory and back out again afterwards.